client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -581,8 +618,16 @@ class WpcomChecklistComponent extends PureComponent {
 	};
 
 	renderSiteLaunchedTask = ( TaskComponent, baseProps, task ) => {
-		const { needsVerification, siteIsUnlaunched, translate } = this.props;
+		const { needsVerification, needsDomainVerification, siteIsUnlaunched, translate } = this.props;
 		const disabled = ! baseProps.completed && needsVerification;
+		let noticeText;
+		if ( disabled ) {
+			if ( needsDomainVerification ) {
+				noticeText = translate( 'Verify your domain before launching your site.' );
+			} else if ( needsDomainVerification ) {
```

Comment:
Should this be `needsEmailVerification` or `needsVerification`? As written, this block is unreachable. 😄 (Note that as-is, `needsEmailVerification` is not being pulled out of the props.)